docs/workflow/testing/libraries/testing-wasm.md

```diff
@@ -7,14 +7,14 @@ In order to be able to run tests, the following JavaScript engines should be ins
 
 They can be installed as a part of [jsvu](https://github.com/GoogleChromeLabs/jsvu).
 
-Please make sure that a JavaScript engine binary is available via command line,  
+Please make sure that a JavaScript engine binary is available via command line,
 e.g. for V8:
```

Comment:
Do you want these to be in the same line or separate lines?

Before removing the spaces they were separate lines, after removing they are in the same line.

Reply:
This looks fine.